index.src.html

```diff
@@ -2089,12 +2089,31 @@ <h3 id="security-cross-origin-leakage">Cross-origin Credential Leakage</h3>
       MUST NOT offer credentials to an origin in response to
       {{CredentialsContainer/get()}} without user mediation if the credential's
       origin is not an exact match for the calling origin. That is,
-      {{Credential}} objects for <code>https://example.com</code> would not be
-      returned directly to <code>https://www.example.com</code>, but could be
+      {{Credential}} objects for <code>https://example.com/</code> would not be
+      returned directly to <code>https://www.example.com/</code>, but could be
       offered to the user via the chooser.
     </li>
+    <li>
+      MAY store credentials without user mediation when
+      {{CredentialsContainer/store()}} is called if the credential was
+      retrieved via a user mediated {{CredentialsContainer/get()}} from a
+      PSL-matched domain and new domain is still PSL-matching the original
```

Comment:
I think we could be a little more precise. How about something like: "`... via a user-mediated {{CredentialsContainer/get()}}, if the credential's {{[[origin]]}}'s <a>registerable domain</a> is the same as the <a>current settings object</a>'s <a for="environment settings object">origin</a>'s <a>registerable domain</a>.`"